### ISSUE TITLE
Daily .tar.gz build should bundle necessary libs

### ISSUE THREAD

**Issue by qwertychouskie** [cubosphere/cubosphere-code#43]
@cddepppp256 gets this error when running the bin:
```
 ./cubosphere: error while loading shared libraries: libPocoZip.so.60: cannot open shared object file: No such file or directory
```
The libs needed by Cubosphere should either be bundled or statically linked to avoid these types of issues.

**Comment by v1993** [comment]
I don't think this static linkage is a good idea. Cubosphere rely on CMake for library search and it doesn't provide required tools.

As for bundling libs, it is better option but I still dislike it. Should we include libc? libstdc++? SDL2? SDL2_Image? This is complicated choice which can't be made easily and automated until you decide to include EVERY used lib into bundle.
